partner_industry_secondary/migrations/11.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_column_renames = {
+    'res_partner': [('sector_id', None)],
+}
+
+_field_adds = [
```

Comment:
This is not needed. When you update the module, Odoo will add these fields from the models definition.

Reply:
Removed.